react-components/src/components/Checkout/receipt.js

```diff
@@ -38,11 +31,14 @@ const Receipt = () => {
                 <div className={classes.textBlock}>
                     {/* prettier-ignore */}
                     <Trans i18nKey="checkout:order-confirmation">
-                        The order number is {{ orderId : order.order_id }}. You will receive an order confirmation email with order
+                        The order number is {{ orderId : orderId }}. You will receive an order confirmation email with order
                         status and other details.
                     </Trans>
                 </div>
-                <Trigger action={continueShopping}>
+                <Trigger
+                    action={async () => {
```

Comment:
Why don't you call `continueShopping()` directly? The `async` function you pass is asynchrone the same way like `continueShopping()`.

Reply:
No reason, I think it's a leftover from previous tests. I'll call it directly